examples/fig10_candlestick.nim

```diff
@@ -0,0 +1,21 @@
+import plotly
+import chroma
+import random
```

Comment:
The `random` import isn't necessary (no big deal of course).

Reply:
Well spotted! Another spurious line from my initial cut and paste ;)